src/index.html

```diff
@@ -22,6 +22,404 @@
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__review-count">Reviews: 5</p>
+          </div>
+
+          <div class="card__price-container">
+            <p class="card__price-label">Price:</p>
+            <p class="card__price">$2,199</p>
+          </div>
+
+          <a
+            class="card__buy"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card">
+          <img
+            class="card__image"
+            src="/src/images/imac.jpeg"
+            alt="iMac A1419"
+          />
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__review-count">Reviews: 5</p>
+          </div>
+
+          <div class="card__price-container">
+            <p class="card__price-label">Price:</p>
+            <p class="card__price">$2,199</p>
+          </div>
+
+          <a
+            class="card__buy"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card">
+          <img
+            class="card__image"
+            src="/src/images/imac.jpeg"
+            alt="iMac A1419"
+          />
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__review-count">Reviews: 5</p>
+          </div>
+
+          <div class="card__price-container">
+            <p class="card__price-label">Price:</p>
+            <p class="card__price">$2,199</p>
+          </div>
+
+          <a
+            class="card__buy"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
```

Comment:
The card content is repeated multiple times with the same data. Consider using a templating system or JavaScript to dynamically generate this content if the data is meant to be the same, or ensure that each card has unique content if they are supposed to represent different items.